pydicom/encaps.py

```diff
@@ -370,3 +379,207 @@ def read_item(fp):
 
     item_data = fp.read(length)
     return item_data
+
+
+# Functions for encapsulating data
+def fragment_frame(frame, no_fragments=1):
+    """Yield one or more fragments from `frame`.
```

Comment:
Maybe `nr_fragments`? Otherwise it reads like a boolean "no fragments" (at least to me).

Reply:
OK